### ISSUE TITLE
Custom Dataset format

### ISSUE THREAD

**Issue by arita37** [felixleopoldo/benchpress#34]
Hello,

if we wan to use on our own dataset (tabular csv) 
format.

what will be the format of the dataset ?

thx

**Comment by felixleopoldo** [comment]
Hi,

It is CSV files still, where the first line tells the labels of the variables. For multinomial data, the second line should contain the number of states for the variables.

Have a look at [this](https://raw.githubusercontent.com/felixleopoldo/benchpress/master/resources/data/mydatasets/2005_sachs_2_cd3cd28icam2_log_std.csv) example for continuous data  and [this](https://raw.githubusercontent.com/felixleopoldo/benchpress/master/resources/data/mydatasets/2019_olsson_pavlenko_rios_p15_n1000.csv) example for multinomial data.

Just let me know if you run into trouble or have any questions

**Comment by arita37** [comment]
Sure, Thx.

Is there any pre-processing required
for a new dataset ?

Any constraints on the format requirements ?

Thx

**Comment by felixleopoldo** [comment]
For continuous data, it is [recommended](https://arxiv.org/abs/2102.13647) to standardize first. For multinomial data, some algorithms may require enumeration of e.g. k states from 0-k-1. Also, have a look at the Benchpress [paper](https://arxiv.org/abs/2107.03863) for a further description of the data format. 
I hope this could help you

**Comment by arita37** [comment]
Sure, thanks
The columns name should be in the json format ?

**Comment by felixleopoldo** [comment]
No, it should be a CSV file, just have to look into the other data files [here](https://github.com/felixleopoldo/benchpress/tree/master/resources/data/mydatasets) and you will see the structure.
[Here](https://github.com/felixleopoldo/benchpress/blob/master/config/paper_sachs.json) is a real data JSON config example that you may also look at. If you don't know the true graph, graph_id should be set to null.
